src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```diff
@@ -79,6 +79,11 @@ jwt.scanner.server.vulnerability.signatureAttack.nullByte.desc=Signature bytes a
 jwt.scanner.server.vulnerability.signatureAttack.nullByte.refs=http://projects.webappsec.org/w/page/13246949/Null%20Byte%20Injection
 jwt.scanner.server.vulnerability.signatureAttack.nullByte.soln=Validate entire Signature and ensure the validating library or function does process beyond (or ignoring) null bytes.
 
+jwt.scanner.server.vulnerability.signatureAttack.incorrectSignature.name=JWT signature not verified
+jwt.scanner.server.vulnerability.signatureAttack.incorrectSignature.desc=Some implementation fail to properly verify the signature of JWT token, which can result in them accepting tokens with invalid signatures.
```

Comment:
I believe implementation should have an s making it plural. Same with “JWT token”.

Reply:
Updated